src/components/creator/CreatorWizard.tsx

```diff
@@ -323,44 +336,66 @@ const CreatorWizard = ({
 
   return (
     <CreatorWizardContext.Provider value={context}>
-      <FormRenderer
-        onSubmit={() => {}}
-        schema={schema}
-        componentMapper={componentMapper}
+      <Tabs
```

Comment:
@JetyAdam FWIW now would be a great time to start adding some component tests for the tool. It looks like @apinkert added some for help panel already

Reply:
Let's move in smaller steps. The CreatorWizard can be tested during tech debt.